src/League/ApplicationConfigServiceProvider.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace TomPHP\ConfigServiceProvider\League;
+
+use League\Container\ServiceProvider\AbstractServiceProvider;
+use TomPHP\ConfigServiceProvider\ApplicationConfig;
+use TomPHP\ConfigServiceProvider\ApplicationConfigIterator;
+
+final class ApplicationConfigServiceProvider extends AbstractServiceProvider
+{
+    /**
+     * @var ApplicationConfig
+     */
+    private $config;
+
+    /**
+     * @var string
+     */
+    private $prefix;
+
+    /**
+     * @param ApplicationConfig $config
+     * @param string            $prefix
+     */
+    public function __construct(ApplicationConfig $config, $prefix)
+    {
+        $this->prefix   = $prefix;
+        $this->config   = $config;
+        $this->provides = array_map(
+            function ($key) {
+                return $this->keyPrefix() . $key;
+            },
+            $config->getKeys()
+        );
+    }
+
+    public function register()
+    {
+        $container = $this->getContainer();
```

Comment:
@tomphp 

Following https://github.com/tomphp/config-service-provider/pull/28/files/c116b2bd76dd964950e22ff3c899378ad5a2ca9b#r76632437, this is currently not in use, is it?